ifacemonitor/ifacemonitor_test.go

```diff
@@ -347,6 +350,15 @@ var _ = Describe("ifacemonitor", func() {
 		<-nl.userSubscribed
 	})
 
+	It("should skip netlink address updates for ipvs", func() {
+		SetDefaultConsistentlyDuration(1 * time.Second)
```

Comment:
I think the code should have "1s" on the specific Consistently statements for which this is appropriate, instead of setting this default for the whole test suite.  (Unless there is some way of making this change temporarily for a group of tests, using BeforeEach and AfterEach?)